src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -63,38 +63,32 @@ const NavHeader = (props) => {
 
   const AnimatedBlurView = Animated.createAnimatedComponent(BlurView);
 
-  const additionalHeaderHeight = isHeaderHide ? dimen.normalizeDimen(12) : 0;
+  const headerHeight = isHeaderHide
+    ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2
+    : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT;
 
   return (
     <View>
-      <View style={{}}>
+      <View>
         <StatusBar barStyle="light-content" />
         <View
-          style={[
-            styles.navContainer(isHeaderHide),
-            {
-              height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER + additionalHeaderHeight,
-              backgroundColor: COLORS.almostBlack,
-              borderBottomWidth: isHeaderHide ? 1 : 0,
-              borderBottomColor: COLORS.gray310
-            }
-          ]}
+          style={[styles.navContainer(isHeaderHide, headerHeight)]}
           imageStyle={{opacity: isHeaderHide ? 0 : 1}}>
           {initialData?.coverImage === undefined && isLoading ? (
-            <Shimmer width={displayWidth} height={dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER} />
+            <Shimmer width={displayWidth} height={headerHeight} />
           ) : (
             <>
               <View
                 style={{
                   width: '100%',
-                  height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
+                  height: headerHeight
                 }}>
                 <FastImage
                   source={
                     initialData?.coverImage ? {uri: initialData.coverImage} : {uri: coverPath}
                   }
                   style={{
-                    height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
+                    height: headerHeight
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for improving readability and maintainability. The introduction of the `headerHeight` variable simplifies the logic and makes it easier to understand. The styles have been refactored for consistency, which is a good practice. However, I noticed that the `FastImage` component's source prop is being set with a ternary operator checking if `initialData?.coverImage` exists. This could potentially lead to an issue if `initialData` is undefined. To avoid potential errors, you should add an additional check to ensure `initialData` is not undefined.
```diff
- initialData?.coverImage ? {uri: initialData.coverImage} : {uri: coverPath}
+ initialData && initialData.coverImage ? {uri: initialData.coverImage} : {uri: coverPath}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->